tests/decompile-test/cases/sprite_kind.ts

```diff
@@ -0,0 +1,6 @@
+namespace SpriteKind {
+    export const Rocket = SpriteKind.create();
+    export let Whatever = SpriteKind.create();
```

Comment:
Should it allow ``let``s? Currently it fails if ``SpriteKind.Whatever`` is reassigned outside the namespace (probably should never happen in real code, but who knows?):

```typescript
namespace SpriteKind {
    export let Whatever = SpriteKind.create()
}
SpriteKind.Whatever = 1
```

Reply:
Good catch